label_studio/data_manager/actions/basic.py

```diff
@@ -70,6 +70,40 @@ def delete_tasks(project, queryset, **kwargs):
     return {'processed_items': count, 'reload': reload,
             'detail': 'Deleted ' + str(count) + ' tasks'}
 
+def delete_my_tasks_annotations(project, queryset, **kwargs):
+    """ Delete annotations of connected user by tasks ids
+
+    :param project: project instance
+    :param queryset: filtered tasks db queryset
+    """
+    request = kwargs['request']
```

Comment:
It's better to avoid code duplication, if you want to reuse this code just make it as external function above and use in these two places.

Reply:
Thank you @makseq! I changed the code